x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/settings_flyout/schedule/logic/translations.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+// Attack discovery
+
+export const GET_ATTACK_DISCOVERY_SCHEDULE_FAILURE = i18n.translate(
+  'xpack.securitySolution.attackDiscovery.schedule.getScheduleFailDescription',
+  {
+    defaultMessage: 'Failed to fetch attack discovery schedule',
+  }
+);
+
+export const FIND_ATTACK_DISCOVERY_SCHEDULE_FAILURE = i18n.translate(
+  'xpack.securitySolution.attackDiscovery.schedule.findScheduleFailDescription',
+  {
+    defaultMessage: 'Failed to fetch attack discovery schedules',
```

Comment:
nit: consider using the `plural` pattern here too

Reply:
done in https://github.com/elastic/kibana/pull/217917/commits/38585fbef19ecfdf1ef2764d67edeead4f00329e